test/cypress/e2e/login.spec.cy.js

```diff
@@ -0,0 +1,220 @@
+describe('Login page', () => {
+  context('desktop', () => {
+    beforeEach(() => {
+      cy.visit('/#/login');
+      cy.viewport('macbook-16');
+    });
+
+    it('renders logo', () => {
+      cy.dataCy('logo').should('be.visible');
+
+      cy.dataCy('logo').invoke('height').should('be.equal', 80);
+    });
+
+    it.only('renders help button', () => {
+      let config;
+      cy.window().should('have.property', 'i18n');
+      cy.window().then((win) => {
+        config = JSON.parse(win.process.env.RIDE_TO_WORK_BY_BIKE_CONFIG);
+      });
+      cy.window().then(() => {
+        cy.dataCy('button-help').should('be.visible');
```

Comment:
Same as my comment above https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/51/files#r1364873399.

Reply:
Fixed.